torchvision/transforms/transforms.py

```diff
@@ -1483,7 +1488,7 @@ def __repr__(self):
 
 
 class RandomErasing(torch.nn.Module):
-    """ Randomly selects a rectangle region in an image and erases its pixels.
+    """ Randomly selects a rectangle region in an torch Tensor image and erases its pixels.
```

Comment:
Good catch. Perhaps be even more specific and mention that it does not support PIL?